modules/ROOT/pages/deploy-to-runtime-fabric.adoc

```diff
@@ -72,17 +72,63 @@ For newly deployed applications, the status initially displays as *Starting*. Th
 
 To test inbound traffic for deployed applications, you can send a request using the controller IP address along with a host header set to the domain. The host header depends on the structure of the application URL.
 
-* An application URL with the format of _{app-name}.domain.com_ uses the following request header format: `Host: {app-name}.domain.com`. Here’s an example cURL command to verify:
+. Determine the URL path (if it exists) for the application by locating the worker node for the application, for example:
```

Comment:
I suggest: Determine the URL path, if any, for the application by locating the worker node for the application. For example:

I know Kim doesn't like us using "For example" on its own, but in this case, I think it makes the sentence too long.